portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -97,19 +102,44 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Constructs comment and gets comment sentiment
     String commentText = getParameter(request, "comment-text", "");
+    float commentSentiment = getSentiment(commentText);
+
+    Comment comment = new Comment(commentText);
+    comment.setSentimentScore(commentSentiment);
```

Comment:
nit: ideally the sentiment could be passed in the constructor

Reply:
Will do in separate PR. Although I want to keep the sentiment optional, this is still a good change.